pilz_robot_programming/package.xml

```diff
@@ -16,23 +16,22 @@
   <run_depend>moveit_commander</run_depend>
   <run_depend>tf</run_depend>
   <run_depend>rospy</run_depend>
-  <run_depend>pilz_trajectory_generation</run_depend>
+  <run_depend>pilz_industrial_motion_planner</run_depend>
   <run_depend>pilz_msgs</run_depend>
-  <run_depend>python-psutil</run_depend>
+  <run_depend>python3-psutil</run_depend>
```

Comment:
Maybe separate the python3 changes from the moveit-merge, to allow an easier backport to melodic?

Reply:
Yes ...